disk/assets/service_checks.json

```diff
@@ -12,6 +12,6 @@
             "device"
         ],
         "name": "Disk read_write",
-        "description": "Returns `CRITICAL` if filesystem is in read-only mode."
+        "description": "Returns `CRITICAL` if filesystem is in read-only mode. **Note**: This service check is not enabled by default and must be enabled under `service_check_rw` in the [disk.d/conf.yaml][3] config file."
```

Comment:
I haven't seen a link in a service check before, so I just want to verify that having `[disk.d/conf.yaml][3]` actually links to the actual conf.yaml link?

Reply:
@yzhan289 Not entirely sure here but this is the same format used in the doc where this change would be populated: https://github.com/DataDog/integrations-core/blob/master/disk/README.md?plain=1#L15